cl/alerts/management/commands/cl_send_alerts.py

```diff
@@ -37,6 +33,7 @@
 # Only do this number of RT items at a time. If there are more, they will be
 # handled in the next run of this script.
 MAX_RT_ITEM_QUERY = 1000
+HITS_LIMIT = 20
```

Comment:
Can we use the `SCHEDULED_ALERT_HITS_LIMIT` setting to avoid hardcoding this value?

Reply:
Yeah, now `SCHEDULED_ALERT_HITS_LIMIT ` is also equals to `20`, changed it.